backend/src/database/entities/bcws/bcws-personnel-roles.entity.ts

```diff
@@ -28,11 +28,19 @@ export class BcwsSectionsAndRolesEntity {
   })
   expLevel: ExperienceLevel;
 
+  @Column({
+    name: 'rank',
+    type: 'int',
+    nullable: true,
+  })
+  rank: number;
+
   toResponseObject(): BcwsPersonnelRoleRO {
     return {
       role: this.role.name,
       section: this.role.section,
       expLevel: this.expLevel,
+      rank: this.rank,
```

Comment:
if rank is null will this error?

Reply:
It shouldn't; that only happens when we try to extract a value out of a null object